doc/starlight/package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "@astrojs/starlight": "^0.33.1",
     "astro": "^5.5.3",
-    "sharp": "^0.32.5"
+    "sharp": "^0.32.5",
+    "starlight-links-validator": "^0.18.0"
```

Comment:
Shouldn't this be "starlight-image-zoom"?

Reply:
Flabbergasted